main.go

```diff
@@ -4,28 +4,47 @@ import (
 	"fmt"
 	"math/big"
 	"os"
+	"strconv"
 )
 
 var big0 = big.NewInt(0)
```

Comment:
are you still using them?

Reply:
yes, in vscode go fmt runs on every save so it's almost impossible to be sloppy. the default settings have a habit of removing unused imports automatically which is frustrating at times! :smile: